core/shared/platform/windows/win_time.c

```diff
@@ -18,3 +68,126 @@ os_time_get_boot_microsecond()
 
     return ((uint64)ts.tv_sec) * 1000 * 1000 + ((uint64)ts.tv_nsec) / 1000;
 }
+
+uint64
+os_win_clock_res_get(__wasi_clockid_t clock_id, __wasi_timestamp_t *resolution)
+{
+    switch (clock_id) {
+        case __WASI_CLOCK_MONOTONIC:
+        {
+             const __wasi_timestamp_t result =
+                (uint64_t)(((uint64_t)1000000000)
+                           / current_value_of_peformance_counter());
```

Comment:
```suggestion
const uint64 result = (uint64)NANOSECONDS_PER_SECOND / current_value_of_performance_counter();
```

Reply:
But I wonder if this implementation is actually correct? This function is supposed to return return a resolution of the clock, so relying on the performance counter doesn't sound right, as it will change over time.